packages/grafana-ui/src/components/Table/Table.tsx

```diff
@@ -182,7 +182,7 @@ export const Table: FC<Props> = memo((props: Props) => {
         </div>
       );
     },
-    [prepareRow, rows]
+    [data.fields, onCellFilterAdded, prepareRow, rows, tableStyles]
```

Comment:
nit: Best practices is to destruct nested props and use them in the dependencies.

Reply:
done